BootloaderCorePkg/Library/CpuExceptionLib/Ia32/CpuExceptionHandler.c

```diff
@@ -39,6 +41,18 @@ CommonExceptionHandler (
   }
 
   DEBUG ((DEBUG_ERROR, "Exception #%d from 0x%04X:0x%08X !!!\n", ExceptionType, Ptr[1], Ptr[0]));
+
```

Comment:
We have a common function called DumpHex().  
Do you think we can do  DumpHex (0, Ip - 42, 64, Ip - 42) to make it simpler ?  It won't have <> around the current IP, but with the address print, it should be easy to find the related instruction.

Reply:
Sounds good!  I changed it to `DumpHex(0, Ptr[0] - 32, 64, (UINT8 *)(Ptr[0] - 32));`, which gives us a few fewer prologue bytes, but it aligns the faulting instruction to the start of the third line, which makes it easy to find.  I've also updated the commit message to match.

Reply:
Looks good to me. 👍